src/views/Books/Books.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import { fetchBooks } from '../../services/books';
+
+export default function Books() {
+  const [books, setBooks] = useState([]);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const booksData = await fetchBooks();
+        setBooks(booksData);
+      } catch (e) {
+        setError(e.message);
+      }
```

Comment:
Nice use of a try/catch block. Great way to perform error handling on the 3rd party API call.